web/src/gql/queries/group.ts

```diff
@@ -120,3 +121,15 @@ export const MODELS_BY_GROUP = gql`
     }
   }
 `;
+
+export const UPDATE_GROUPS_ORDER = gql`
+  mutation UpdateGroupsOrder($groupIds: [ID!]!) {
+    updateGroupsOrder(input: { groupIds: $groupIds }) {
+      groups {
+        ... on Group {
+          id
+        }
+      }
```

Comment:
I think there is no need to add `... on Group` here
```suggestion
      groups {
        id
      }
```